packages/flutter_tools/test/src/common.dart

```diff
@@ -25,14 +25,7 @@ export 'package:test_core/test_core.dart' hide TypeMatcher, isInstanceOf; // Def
 test_package.TypeMatcher<T> isInstanceOf<T>() => isA<T>();
 
 void tryToDelete(Directory directory) {
-  // This should not be necessary, but it turns out that
-  // on Windows it's common for deletions to fail due to
-  // bogus (we think) "access denied" errors.
-  try {
-    directory.deleteSync(recursive: true);
-  } on FileSystemException catch (error) {
-    print('Failed to delete ${directory.path}: $error');
-  }
+  directory.deleteSync(recursive: true);
```

Comment:
I'd like to leave this for now in case we see flakes around this pop up again.  Should I add a TODO to remove it?

Reply:
The _actual_ reason these files were failing to delete was because we had processes that weren't being killed before trying to delete files in use, which on Windows causes exceptions.

Reply:
Could you link to the Dart SDK bug?

Reply:
I'm not sure if that's the right bug to link here - if anything I'd say file a new bug to track removing this code once we're convinced the current change hasn't re-introduced old flakes.